mlir/lib/Dialect/SCF/Transforms/ParallelLoopTiling.cpp

```diff
@@ -195,6 +195,9 @@ struct ParallelLoopTiling
   }
 
   void runOnOperation() override {
+    for (auto tileSize : tileSizes)
+      if (tileSize == 0)
+        signalPassFailure();
```

Comment:
I would emit an error message here (`mlir::emitError`).

Reply:
I took a stab at this. Is there a way to specify that the location of the error message is from a command line argument? Or is the best way to go forward using `UnknownLoc`?

Reply:
`UnknownLoc` sounds good here.